src/index.html

```diff
@@ -11,12 +11,111 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="moyo.png.png"
+          alt="Logo-MOYO"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
```

Reply:
Still not fixed
<img width="908" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/f8ba6da4-17d7-4850-a08f-26db1f924cad">